lib/utils.pm

```diff
@@ -808,6 +809,22 @@ sub handle_login {
     send_key "ret";
 }
 
+sub handle_logout {
+    # hide mouse for clean logout needles
+    mouse_hide();
+    # logout
+    if (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'lxde')) {
+        my $command = check_var('DESKTOP', 'gnome') ? 'gnome-session-quit' : 'lxsession-logout';
+        x11_start_program("$command");    # opens logout dialog
+        assert_screen 'logoutdialog' unless check_var('DESKTOP', 'gnome');
+    }
```

Comment:
$same = 'I\'m exporting existing code and I don't eager to modify it, but feel free to change it'